pyhealth/medcode/inner_map.py

```diff
@@ -170,11 +172,3 @@ def get_descendants(self, code: str) -> List[str]:
         )
         return descendants
 
-
-if __name__ == "__main__":
-    icd9cm = InnerMap.load("ICD9CM")
-    print(icd9cm.stat())
-    print("428.0" in icd9cm)
-    print(icd9cm.lookup("4280"))
-    print(icd9cm.get_ancestors("428.0"))
-    print(icd9cm.get_descendants("428.0"))
```

Comment:
I think this was an artifact from development

Reply:
Maybe do not delete them as they will be used for testing in future dev.

We are implementing the unittest right now. After unittest, we may delete them.

Reply:
@ycq091044 I put it back. I wrote some unittest for these __repr__ functions, but I'll wait until you release your unittest classes before making a PR for those

Reply:
Great! thanks @BPDanek. We are currently slowly developing the unittest since I am pretty busy handling the large coursera course. If you are familiar with unittest suite and interested in making some solid contribution. Then, we can work together on the unittest module.